lab_1/main.py

```diff
@@ -11,7 +11,13 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
+    if isinstance(text, str):
```

Comment:
put checks with None to the beginning
it'd be better to read 

 if the argument is incorrect, we return None right at the beginning

Reply:
Done